### DISCUSSION TITLE
Here is some of my hardware to use with securitysystem plugin

### DISCUSSION THREAD

**Discussion by splashibiza** [MiguelRipoll23/homebridge-securitysystem#109] [Show and tell]
Still waiting to receive the Philips hue motion sensor and the 12v transformer for the double tone siren

![image](https://user-images.githubusercontent.com/53734931/80572104-fd711800-89fd-11ea-96fa-7aa443320a93.jpg)

**Comment by MiguelRipoll23** [comment]
Looking good! 😲

**Comment by 0ldF0x** [comment]
And here is mine
![00EBB807-0BC4-4318-867F-7F301D0FC205](https://user-images.githubusercontent.com/4620401/80639800-cb45d180-8a62-11ea-95b0-05607db1c922.jpeg)

**Comment by splashibiza** [comment]
@0ldF0x  nice! Is that a custom made homekit siren? What hardware does it involve? I can see some sort of relay?

**Comment by 0ldF0x** [comment]
Yep. 
Bosch sg200 siren, with custom made PCB, ULN2003an transistor chip and Arduino clone with RF (Moteino). All powered by 24v DC power supply.
